gprofiler/merge.py

```diff
@@ -250,9 +254,14 @@ def concatenate_profiles(
 
     for pid, stacks in process_profiles.items():
         container_name = _get_container_name(pid, docker_client, add_container_names)
+        application_name = get_application_name(pid)
+        prefix = container_name + ";" if add_container_names else ""
         for stack, count in stacks.items():
+            if add_application_names and application_name is not None:
+                stack = STACK_COMM_REGEX.sub(application_name, stack)
```

Comment:
nitpick, but I think that just `stack.split(';', 1)` would be faster?

Reply:
You were right...
```
In [14]: def rep1(s, y):
    ...:     return STACK_COMM_REGEX.sub(y, s)

In [21]: def rep2(s, y):
    ...:     return f'{y};{s.split(";", maxsplit=1)[1]}'

In [22]: %timeit rep1(x, "bbb")
30.6 µs ± 51.1 ns per loop (mean ± std. dev. of 7 runs, 10000 loops each)

In [23]: %timeit rep2(x, "bbb")
447 ns ± 5.45 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)
```

Reply:
Not a good enough check because the input data is short (test it with a larger stack).

Anyway I'm sure it'll be faster also with that

Reply:
How do you know the input data is short? 
Input data is `x`

Reply:
Eh you're right, I read this function incorrectly

Reply:
So you're changing that?

Reply:
no pushed.. sorry 😢